packages/malloy/src/lang/parse-malloy.ts

```diff
@@ -783,10 +785,40 @@ export abstract class MalloyTranslation {
     const attempt = this.translateStep.step(this, extendingModel);
     if (attempt.final) {
       this.finalAnswer = attempt;
+      this.buildImports();
     }
     return attempt;
   }
 
+  importAt(position: DocumentPosition): ImportLocation | undefined {
+    // Here we assume that imports DO NOT overlap. And then we do a linear
+    // search to find the one we're looking for.
+    for (let index = 0; index < this.imports.length; index++) {
+      const imp = this.imports[index];
+      if (
+        imp.location.range.start.line <= position.line &&
```

Comment:
Nit: it'd be nice to abstract this logic to a function `locationContainsPosition` or something like that, since this is repeated in the `ReferenceList` class.

Reply:
fixed